### ISSUE TITLE
Close button in group indicator should have a href attribute

### ISSUE THREAD

**Issue by VicTachev** [telerik/kendo-ui-core#5683]
Currently the close button in the group indicator does not have a href attribute. It should be included in order to meet accessibility requiremetns.

![image](https://user-images.githubusercontent.com/11771009/77437918-e54e1d80-6ded-11ea-97c9-fce9a8c32dd4.png)

**Comment by GaloisGirl** [comment]
If it's a *button*, it also needs `role="button"`, or to be render with the `<button>` tag.

**Comment by dmanova** [comment]
As of R3 2020 the ` href="#"` and `data-role="button" ` properties will be added.

**Comment by dmanova** [comment]
Additional properties are needed to make it accessible and will be considered in the future.

**Comment by dmanova** [comment]
Added aria-label="Remove grouping by _FIELD NAME_ field".
